APIs/S3.md

```diff
@@ -12,9 +12,7 @@ To generate the necessary credentials you must have a registered account on data
 
 ## Request for secrets
 
-In order to obtain secrets,<a href= "https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new" target="_blank">Submit a request</a> and provide the e-mail address of the registered Copernicus Data Space Ecosystem user for whom the credentials will be generated.
-
-
+In order to obtain secrets, create a ticket to CDSE support. Our operators will provide you with the appropriate credentials. You must provide in such request the e-mail address of the registered CDSE user for whom the credentials will be generated.
```

Comment:
why is the link removed? this seems like a useful pointer

Reply:
not sure what was pulled to avoid conflict